features/automation/controls/AdjustSlFormControl.tsx

```diff
@@ -100,24 +113,30 @@ export function AdjustSlFormControl({
   type Action =
     | { type: 'stop-loss'; stopLoss: BigNumber }
     | { type: 'close-type'; toCollateral: boolean }
+    | { type: 'isEditing'; isEditing: boolean }
 
   function reducerHandler(state: AddFormChange, action: Action): AddFormChange {
     switch (action.type) {
       case 'stop-loss':
         return { ...state, selectedSLValue: action.stopLoss }
       case 'close-type':
         return { ...state, collateralActive: action.toCollateral }
+      case 'isEditing':
+        return { ...state, isEditing: action.isEditing }
     }
   }
 
   const token = vault.token
   const tokenData = getToken(token)
   const currentCollateralData = collateralPrice.data.find((x) => x.token === vault.token)
   const ethPrice = collateralPrice.data.find((x) => x.token === 'ETH')?.currentPrice!
-  const startingSlRatio = isStopLossEnabled ? stopLossLevel : ilkData.liquidationRatio
-  const isEditing = isStopLossEnabled
-    ? !selectedSLValue.eq(stopLossLevel.multipliedBy(100))
-    : !selectedSLValue.eq(ilkData.liquidationRatio.multipliedBy(100))
+  const startingSlRatio = isStopLossEnabled
+    ? stopLossLevel
+    : new BigNumber(
+        ilkData.liquidationRatio.plus(vault.collateralizationRatio).dividedBy(2).toFixed(2, 2),
```

Comment:
could you please use the enum as a 2nd argument for `toFixed` method. kinda difficult to memorize all the variants 
```suggestion
        ilkData.liquidationRatio.plus(vault.collateralizationRatio).dividedBy(2).toFixed(2, BigNumber.ROUND_CEIL),
```